slim/defaults/configs/znc.conf

```diff
@@ -0,0 +1,61 @@
+// WARNING
+//
+// Do NOT edit this file while ZNC is running!
+// Use webadmin or *controlpanel instead.
+//
+// Altering this file by hand will forfeit all support.
+//
+// But if you feel risky, you might want to read help on /znc saveconfig and /znc rehash.
+// Also check http://en.znc.in/wiki/Configuration
+
+AnonIPLimit = 10
+ConnectDelay = 5
+HideVersion = false
+LoadModule = webadmin
+MaxBufferSize = 500
+ProtectWebSessions = false
+SSLCertFile = /znc-data/znc.pem
+ServerThrottle = 30
+Version = 1.6.5
+
+<Listener listener0>
+	AllowIRC = true
+	AllowWeb = true
+	IPv4 = true
+	IPv6 = false
+	Port = 8080
+	SSL = true
+	URIPrefix = /
+</Listener>
+
+<User admin>
+	Admin = true
+	Allow = *
+	AltNick = Nick_
+	AppendTimestamp = false
+	AutoClearChanBuffer = true
+	AutoClearQueryBuffer = true
+	Buffer = 50
+	DenyLoadMod = false
+	DenySetBindHost = false
+	Ident = Nick
+	JoinTries = 10
+	LoadModule = chansaver
+	LoadModule = controlpanel
+	MaxJoins = 0
+	MaxNetworks = 1
+	MaxQueryBuffers = 50
+	MultiClients = true
+	Nick = Nick
+	PrependTimestamp = true
+	QuitMsg = %znc%
+	RealName = Real Name
+	StatusPrefix = *
+	TimestampFormat = [%H:%M:%S]
+
+	<Pass password>
+		Hash = cc6b37824eeda0b0ec028e18e98684ec71a3dbef57671f7c8d70af386455149e
+		Method = SHA256
+		Salt = hkXd849mvtdBhHgCQHIr
```

Comment:
As far as I know, ZNC still supports unhashed passwords. That make sense to be use here. However, no idea what the state of that support is. It should only exist for backwards compatibility....